inc/views/base/newreply/newreply.twig

```diff
@@ -0,0 +1,263 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ trans('post_reply_to', thread.subject) }}</title>
+    <script type="text/javascript" src="{{ asset_url('jscripts/post.js') }}"></script>
```

Comment:
We should probably add a new `block` to the master layout to put JS at the end, before the closing `</body>`. I'll do that at a later date and move things like this to that block.